guides/deploy/starter_deploy_kubernetes.html

```diff
@@ -37,7 +37,7 @@ <h1 class="header-title">KITURA</h1>
       <h1>DEPLOY YOUR APP USING KUBERNETES AND DOCKER</h1>
     </div>
 
-<p>This guide will show you how to deploy the Kitura application you created by following our <a href="https://www.kitura.io/guides/gettingstarted">Getting Started</a> guide to a <a href="https://kubernetes.io/">Kubernetes</a> based environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud and <a href="https://console.bluemix.net/docs/containers/container_index.html#container_index">create a Kubernetes cluster</a> for free, or <a href="https://kubernetes.io/docs/setup/minikube/">set one up locally using minikube</a>.</p>
+<p>This guide will show you how to deploy the Kitura application you created by following our <a href="guides/gettingstarted.html">Getting Started</a> guide to a <a href="https://kubernetes.io/">Kubernetes</a> based environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud and <a href="https://console.bluemix.net/docs/containers/container_index.html#container_index">create a Kubernetes cluster</a> for free, or <a href="https://kubernetes.io/docs/setup/minikube/">set one up locally using minikube</a>.</p>
```

Comment:
Don't we need a capital M for minikube? The Getting Started link needs fixing here too.